pallets/gear/src/lib.rs

```diff
@@ -190,7 +189,7 @@ pub mod pallet {
         type CodeStorage: CodeStorage;
 
         /// Implementation of a storage for programs.
-        type ProgramStorage: PausedProgramStorage<
```

Comment:
@gshep are you agree with this solution?

Reply:
yep